datalad_fuse/fuse_.py

```diff
@@ -163,7 +163,10 @@ def getattr(self, path, fh=None):
                         fsspec_file, timestamp=self._adapter.get_commit_datetime(path)
                     )
                 if to_close:
-                    fsspec_file.close()
+                    try:
+                        fsspec_file.close()
+                    except KeyError:
+                        pass
```

Comment:
FWIW if we are to proceed this way, would be worth adding at least DEBUG level log messages so we have some track of those

Reply:
Done: da59235